tol_lab_share/processors/bioscan_pool_xp_to_traction_processor.py

```diff
@@ -16,41 +17,41 @@
 class BioscanPoolXpToTractionProcessor(BaseProcessor):
     """Processor for consumed messages exporting Pool XP tubes to Traction."""
 
-    def __init__(self, schema_registry: SchemaRegistry, config: Any):
+    def __init__(self, config: Any):
         """Prepare the initial internal state of the processor.
 
         Args:
-            schema_registry (SchemaRegistry): a schema registry for message parsing.
             config (Any): main configuration for the app.
         """
         LOGGER.debug("BioscanPoolXpToTractionProcessor::__init__")
 
-        self._schema_registry = schema_registry
         self._config = config
 
     @staticmethod
     def instantiate(
-        schema_registry: SchemaRegistry, _: BasicPublisher, config: Any
+        _schema_registry: SchemaRegistry, _basic_publisher: BasicPublisher, config: Any
     ) -> "BioscanPoolXpToTractionProcessor":
         """Instantiate a BioscanPoolXpToTractionProcessor."""
-        return BioscanPoolXpToTractionProcessor(schema_registry, config)
+        return BioscanPoolXpToTractionProcessor(config)
```

Comment:
I assume the `schema_registry` and `basic_publisher` aren't used yet because they their use is coming up in a different story?

Reply:
They are used in other processors, not this one.  This processor doesn't generate feedback messages via RabbitMQ and you only need those two things if you're going to be doing that.